app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -34,12 +43,30 @@ export default Controller.extend({
       .loadItems({ q: this.searchTerm })
       .then(response => this._processLoadResponse(response))
       .catch(() => this._bookmarksListDenied())
-      .finally(() =>
+      .finally(() => {
         this.setProperties({
           loaded: true,
           loading: false
-        })
-      );
+        });
+
+        // TODO(martin): This should be pulled out into a bookmark-list component,
+        // the controller is not the best place for this.
+        if (!this.boundClick) {
+          setTimeout(() => {
```

Comment:
we don't use setTimeout as much as possible, in your case `schedule("afterRender", ...` is probably the best choice